lib/Input.js

```diff
@@ -0,0 +1,24 @@
+const minimist = require('minimist');
+class Input {
+
+  parse(arr) {
```

Comment:
This `parse` works here, but putting this in a `constructor()` would make this cleaner and allow a new instance of the `Input` class to parse on declaration. 

```js
constructor(arr){
const obj = minimist(arr);
    delete obj._;
    let [[type, payload]] = Object.entries(obj);
    if(type === 'a') type = 'add';
this.type = type;
this.payload = payload;
}
```